### PR TITLE
Rugged begone

### PR DESCRIPTION
**Pull request by msmirnoff**
Removed rugged - using it would leave the repository working dir in a messed up state even though the changes were committed.

### CODE REVIEW

File: lib/deploy/repository.rb [hunk 1]
```diff
@@ -21,75 +12,53 @@ def prepare!(tag)
   private
 
   def tag_exists?
-    repo.tags.map(&:name).include? @tag
+    require 'open3'
```
Comment (alxberardi) [inline lib/deploy/repository.rb]: How many times do we call this method? I wonder if it could be nicer to put requires at the top of the file

**Comment by alxberardi** [comment]
One minor comment you might want to address, :+1: :100: beside that! :)

**Comment by msmirnoff** [comment]
@AlessandroBerardi 
Addressed.

**Comment by alxberardi** [comment]
Thanks @msmirnoff :+1: :+1: :+1:

**Pull request merged**